### PR TITLE
Metadata and config for iof

### PR DESCRIPTION
**Pull request by mereolog**
This sets up infrastructure for [IOF viewer](https://spec.industrialontologies.org/iof/ontology).

### CODE REVIEW

File: etc/onto-viewer-web-app/config/ontology_config.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+ontologies:
+  source:
+    - url: 'https://spec.industrialontologies.org/ontology/AboutIOFDev/'
```
Comment (przemekgradzki) [inline etc/onto-viewer-web-app/config/ontology_config.yaml:3]:
```suggestion
    - url: 'https://spec.industrialontologies.org/ontology/AboutIOFDev/'
    - url: 'https://spec.industrialontologies.org/ontology/MetadataIOF/'
```

File: etc/onto-viewer-web-app/config/ontology_config.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  catalog_path:
+    - ontologies/catalog-v001.xml
```
Comment (przemekgradzki) [inline etc/onto-viewer-web-app/config/ontology_config.yaml:6]:
```suggestion
```

Not required after ['zip' set as 'URL#fragment' #121](https://github.com/edmcouncil/ontology-publisher/pull/121)

Comment (mereolog) [reply etc/onto-viewer-web-app/config/ontology_config.yaml:6]: Let's us leave it for now.

**Review by przemekgradzki — APPROVED** [review]

File: MetadataIOF.rdf [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	xmlns:sm="http://www.omg.org/techprocess/ab/SpecificationMetadata/"
+	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
+	
+	<owl:Ontology rdf:about="https://spec.industrialontologies.org/ontology/MetadataIOF/">
+		<rdfs:label>Metadata for the IOF Ontology</rdfs:label>
+		<dct:abstract>The IOF Metadata Ontology describes the various modules that make up the IOF ontology library.</dct:abstract>
+		<dct:issued rdf:datatype="&xsd;dateTime">2023-01-20T00:00:00</dct:issued>
+		<dct:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dct:license>
+		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/Core/Metadatacore/"/>
+		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/maintenance/Metadatamaintenance/"/>
```
Comment (przemekgradzki) [inline MetadataIOF.rdf:36]:
```suggestion
		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/maintenance/Metadatamaintenance/"/>
		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/supplychain/Metadatasupplychain/"/>
```

File: catalog-v001.xml [hunk 4]
```diff
@@ -6,4 +6,7 @@
     <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/" uri="./core/meta/AnnotationVocabulary.rdf"/>
     <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/core/MappingCommonsToIOFCore/" uri="./core/commons-to-core-mapping/MappingCommonsToIOF.rdf"/>
     <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/core/meta/MappingAnnotationVocabularyToCommons/" uri="./core/commons-to-core-mapping/meta/MappingAnnotationVocabularyToCommons.rdf"/>
+    <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/core/Core/Metadatacore/" uri="./core/Metadatacore.rdf"/>
```
Comment (przemekgradzki) [inline catalog-v001.xml:9]:
```suggestion
    <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/MetadataIOF/" uri="./MetadataIOF.rdf"/>
    <uri id="User Entered Import Resolution" name="https://spec.industrialontologies.org/ontology/core/Core/Metadatacore/" uri="./core/Metadatacore.rdf"/>
```

**Review by przemekgradzki — CHANGES_REQUESTED** [review]

**Review by przemekgradzki — APPROVED** [review]

**Review by ElisaKendall — APPROVED** [review]

**Review by wsobel — APPROVED** [review]
Looks fine to me. Some pieces of that drive the onto viewer I am not capable of adequately reviewing. In general, content looks ok.

**Comment by chris-will** [comment]
So review and possibly update our process after releasing maintenance, supply chain and core.

**Pull request merged**